### PR TITLE
616 | Upcoming git course details October to November

### PR DESCRIPTION
**Pull request by ns-rse**
Three event pages and a blog post for upcoming courses.

### CODE REVIEW

File: _events/workshop-2022-10-03-git-zero-hero.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Persistent failure to attend booked courses might result in you being excluded from future training opportunities.
+
+<hr/>
+
+More details and registration via
+[eventbrite](https://www.eventbrite.com/e/git-github-through-gitkraken-from-zero-to-hero-registration-406105320947).
+
+
+**Please note:** This workshop will run over 2 half days from 12:30-16:30 on Monday 3rd and Tuesday 4th October 2022.
```
Comment (Robadob) [inline _events/workshop-2022-10-03-git-zero-hero.md]: The linked eventbrite above has this date/time

Date and time
Mon, 3 Oct 2022, 09:30 –

Tue, 4 Oct 2022, 13:00 BST

Comment (Robadob) [reply _events/workshop-2022-10-03-git-zero-hero.md]: (is it 9-1 each day or 12:30-16:30?)

Comment (Robadob) [reply _events/workshop-2022-10-03-git-zero-hero.md]: From description

2022-10-03 09:30 to 13:00
2022-10-04 09:30 to 13:00

File: _events/workshop-2022-10-31-git-zero-hero.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Persistent failure to attend booked courses might result in you being excluded from future training opportunities.
+
+<hr/>
+
+More details and registration via
+[eventbrite](https://www.eventbrite.com/e/git-github-through-gitkraken-from-zero-to-hero-registration-406112592697).
+
+
+**Please note:** This workshop will run over 2 half days from 12:30-16:30 on Monday 31st October and Tuesday 1st November 2022.
```
Comment (Robadob) [inline _events/workshop-2022-10-31-git-zero-hero.md]: Again, link says its morning not afternoon.

File: _events/workshop-2022-11-28-git-zero-hero.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+Persistent failure to attend booked courses might result in you being excluded from future training opportunities.
+
+<hr/>
+
+More details and registration via
+[eventbrite](https://www.eventbrite.com/e/git-github-through-gitkraken-from-zero-to-hero-registration-406105320947).
+
+
+**Please note:** This workshop will run over 2 half days from 12:30-16:30 on Monday 28th and Tuesday 29th June 2022.
```
Comment (Robadob) [inline _events/workshop-2022-11-28-git-zero-hero.md]: This date has already passed

File: _events/workshop-2022-11-28-git-zero-hero.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+Running courses that are not fully attended wastes our funding (which is provided by taxpayers, charities and students,
+amongst others) and reduces our collective capacity to improve research outputs and researcher experiences.
+
+
+Persistent failure to attend booked courses might result in you being excluded from future training opportunities.
+
+<hr/>
+
+More details and registration via
+[eventbrite](https://www.eventbrite.com/e/git-github-through-gitkraken-from-zero-to-hero-registration-406105320947).
```
Comment (Robadob) [inline _events/workshop-2022-11-28-git-zero-hero.md]: This link matches the one in file `_events/workshop-2022-10-03-git-zero-hero.md`

Comment (Robadob) [reply _events/workshop-2022-11-28-git-zero-hero.md]: Should link to the november event?

File: _events/workshop-2022-11-28-git-zero-hero.md (hunk 3 above)
Comment (Robadob) [reply _events/workshop-2022-11-28-git-zero-hero.md]: I also expect it's supposed to be a morning session

**Review by Robadob — CHANGES_REQUESTED** [review]
See individual comments, alot of issues.

File: _events/workshop-2022-10-03-git-zero-hero.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
+---
+title: "git & GitHub through GitKraken - from Zero to Hero!"
+category: gitzerohero
+tags: collaboration, git, github, gitkraken
+permalink: /training/workshop/2022-06-06-git-zero-hero
```
Comment (davidwyld) [inline _events/workshop-2022-10-03-git-zero-hero.md]:
```suggestion
permalink: /training/workshop/2022-10-03-git-zero-hero
```

File: _events/workshop-2022-10-31-git-zero-hero.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
+---
+title: "git & GitHub through GitKraken - from Zero to Hero!"
+category: gitzerohero
+tags: collaboration, git, github, gitkraken
+permalink: /training/workshop/2022-06-06-git-zero-hero
```
Comment (davidwyld) [inline _events/workshop-2022-10-31-git-zero-hero.md]:
```suggestion
permalink: /training/workshop/2022-10-31-git-zero-hero
```

File: _events/workshop-2022-11-28-git-zero-hero.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
+---
+title: "git & GitHub through GitKraken - from Zero to Hero!"
+category: gitzerohero
+tags: collaboration, git, github, gitkraken
+permalink: /training/workshop/2022-06-06-git-zero-hero
```
Comment (davidwyld) [inline _events/workshop-2022-11-28-git-zero-hero.md]:
```suggestion
permalink: /training/workshop/2022-11-28-git-zero-hero
```

**Review by davidwyld — CHANGES_REQUESTED** [review]
Looks good! Cheers Neil! Jst a couple of small corrections + Rob's ones.

File: _events/workshop-2022-10-03-git-zero-hero.md (hunk 1 above)
Comment (ns-rse) [reply _events/workshop-2022-10-03-git-zero-hero.md]: Yes, one of the short-comings of Eventbrite, if its not a single event then it creates two events and people have to sign-up for both. I didn't want to confuse people and have them think that they only had to attend one session and have people signing up for one day and not the other.

But that isn't the point you're making, I didn't change the text to reflect morning sessions, thank you.

File: _events/workshop-2022-10-03-git-zero-hero.md (hunk 5 above)
Comment (ns-rse) [reply _events/workshop-2022-10-03-git-zero-hero.md]: Ta, duly done.

**Review by Robadob — APPROVED** [review]
Looks like everything has been addressed

**Comment by Robadob** [comment]
P.S. FYI if you included `Closes #616` in your commit message (or similar, there are a few keywords), rather than just `616` it would auto close the issue when a commit containing said message was merge.

Likewise, I think if your commit message includes `#616` it will be tagged inside that issue.

**Comment by ns-rse** [comment]
> P.S. FYI if you included `Closes #616` in your commit message (or similar, there are a few keywords), rather than just `616` it would auto close the issue when a commit containing said message was merge.
> 
> Likewise, I think if your commit message includes `#616` it will be tagged inside that issue.

Thanks for the reminder I sometimes remember to use `Resolves #<issue>` but don't always as is the case here.

**Pull request merged**